Formula/pdns.rb

```diff
@@ -35,6 +35,12 @@ class Pdns < Formula
 
   uses_from_macos "curl"
 
+  on_linux do
+    depends_on "gcc" # for C++17
```

Comment:
```suggestion
    depends_on "gcc" => :build # for C++17
```

Reply:
Building with `gcc` usually creates linkage, but we can try it.